fla/ops/generalized_delta_rule/README.md

```
@@ -33,5 +33,69 @@ The second variant is DPLR, where we have:
 Here, $\mathbf{I}$ is replaced by a diagonal matrix $\mathbf{D}_t$. This transition matrix structure has been utilized in RWKV7.
 
 ## Efficient Chunkwise Implementation
+The original [technical note](https://drive.google.com/file/d/1qqc6THTRc2bw-LtwsbGNxNDw00sNzi5M/view?usp=sharing) on chunking DPLR contains minor mathematical inconsistencies. Below, we re-do the computations.
 
-For detailed information about efficient chunkwise implementation, please refer to our [technical note](https://drive.google.com/file/d/1rJbO3dU4fe7OKG3w7Yg058z_BNIuavNF/view?usp=sharing).
+If you have questions about or comments on the below derivations, feel free to reach out: philipp.nazari@tuebingen.mpg.de.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)